src/types/presets.ts

```diff
@@ -65,4 +67,23 @@ export interface PresetOptions {
       maxDuration: number;
     };
   };
+  cloudflarePages: {
```

Comment:
I would prefer if we used `cloudflare.pages.routes` instead of `cloudflarePages.routes`